.travis.yml

```diff
@@ -1,7 +1,8 @@
 ---
 language: node_js
 node_js:
-  - "0.12"
+  - "4"
+  - "6"
```

Comment:
@trentmwillis Good suggestion - I've added `"stable"`.

I think I've understood your `try:each` suggestion correctly, but if you wouldn't mind verifying the last commit in this PR I'd appreciate it!